base/docs/helpdb.jl

```diff
@@ -14567,6 +14567,21 @@ include
 doc"""
 ```rst
 ::
+           include_dependency(path::AbstractString)
+
+In a module, declare that the file specified by `path` (relative or
+absolute) is a dependency for precompilation; that is, the
+module will need to be recompiled if this file changes.
+
+This is only needed if your module depends on a file that is not
+used via `include`.   It has no effect outside of compilation.
+```
+"""
+include_dependency
```

Comment:
Newly added functions do not appear to get correctly populated by `doc/genstdlib.jl`. I think you need to add an opening signature for this somewhere in the RST docs for it to work?